src/analysis/special_functions/trigonometric/angle.lean

```diff
@@ -27,10 +27,19 @@ namespace angle
 instance angle.add_comm_group : add_comm_group angle :=
 quotient_add_group.add_comm_group _
 
+instance : topological_space angle :=
+quotient_add_group.quotient.topological_space _
+
+instance : topological_add_group angle :=
+topological_add_group_quotient _
+
```

Comment:
I think we should be able to drop these if we use `derive` when defining `angle` above.
```suggestion
```
I.e., we would replace `def angle` with:
```lean
@[derive [add_comm_group, topological_space, topological_add_group]]
def angle : Type :=
```

Reply:
I've changed it to use `derive`.